oauthenticator/tests/test_globus.py

```diff
@@ -177,26 +241,22 @@ def test_revoke_tokens(monkeypatch):
     assert logout_handler.clear_login_cookie.called
 
 
-async def test_logout_revokes_tokens(monkeypatch, mock_globus_user):
-
+async def test_logout_revokes_tokens(globus_client, monkeypatch, mock_globus_user):
+    globus_client.add_host('auth.globus.org', [('/v2/oauth2/token/revoke',
+                                               revoke_token_request_handler)])
     authenticator = GlobusOAuthenticator()
-    logout_handler = mock_handler(GlobusLogoutHandler,
-                                  authenticator=authenticator)
-    monkeypatch.setattr(
-        web.RequestHandler,
-        'redirect',
-        Mock()
-    )
+    logout_handler = mock_handler(GlobusLogoutHandler, authenticator=authenticator)
+
+    # Setup
+    monkeypatch.setattr(web.RequestHandler, 'redirect', Mock())
     logout_handler.get_current_user = Mock(return_value=mock_globus_user)
     logout_handler._jupyterhub_user = mock_globus_user
     monkeypatch.setitem(logout_handler.settings, 'statsd', Mock())
     monkeypatch.setitem(logout_handler.settings, 'login_url', '')
 
     logout_handler.clear_login_cookie = Mock()
-    authenticator.revoke_service_tokens = Mock()
     authenticator.revoke_tokens_on_logout = True
 
     await logout_handler.get()
-    assert authenticator.revoke_service_tokens.called
     auth_state = await mock_globus_user.get_auth_state()
-    assert auth_state == {'tokens': ''}
+    assert auth_state == {'tokens': {}}
```

Comment:
Globus Test coverage is now 100%! 

Unfortunately, these changes don't fix the GenericOAuthenticator tests.